lib/active_model/serializer/adapter/attributes.rb

```diff
@@ -24,9 +24,27 @@ def fragment_cache(cached_hash, non_cached_hash)
         private
 
         def serializable_hash_for_collection(options)
+          options.merge!(batch_cache(options))
+
           serializer.map { |s| Attributes.new(s, instance_options).serializable_hash(options) }
         end
 
+        def batch_cache(options)
+          return {} if options[:batch_cache].present? || ActiveModelSerializers.config.cache_store.blank?
+
+          keys = CachedSerializer.object_cache_keys(serializer, @include_tree)
+
+          return {} unless keys.present?
```

Comment:
could be `return {} if keys.blank?` (but could really be `if keys.nil?`, right?)